StylableSwiftUI/Classes/Stylist.swift

```diff
@@ -28,15 +28,18 @@ public class Stylist: ObservableObject {
     @Published private var styles: [Style] {
         didSet {
             self.scoredStyleMatchCache = [:]
+            Self.scoredImageStyleMatchCache = [:]
         }
     }
     @Published public var currentTheme: Theme? {
         didSet {
             self.scoredStyleMatchCache = [:]
+            Self.scoredImageStyleMatchCache = [:]
         }
     }
 
     private var scoredStyleMatchCache: [StylistIdentifier: Style] = [:]
+    static var scoredImageStyleMatchCache: [StylistIdentifier: UIImage] = [:]
```

Comment:
I wonder if storing all images in memory is a bad idea; can we store the identifier instead?